### DISCUSSION TITLE
Is it possible to install x86 packages to be run under rosetta?

### DISCUSSION THREAD

**Discussion by fakedrake** [Homebrew/discussions#769] [Everyday usage] [answered]
Hello and thank you for all the hard work you are putting into homebrew. I see in `brew config` an entry that reads `rosetta 2: false`. I assume that means "only install native bottles". Well, there are some programs (GHC in my case) that are not there yet to be run under apple silicon. Is it possible to tell brew to install them and run them with rosetta?

**Comment by carlocab** [comment, marked as answer]
You need a separate installation of `brew` in `/usr/local` to do that. Run the installer under Rosetta to install a Rosetta version of `brew`.

**Reply by fakedrake** [reply]
Thanks a lot! Just for future reference and to clarify for other people in the same situation. I already had brew in `/opt/homebrew/bin/brew`

```
$ env /usr/bin/arch -x86_64 /bin/zsh
$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

The installer will realize that we are running in x86 mode and install in `/usr/local`. Then depending on which executable we call to install it will get the x86 or the arm64 bottle.
